### ISSUE TITLE
 Island browser support Not available in PlayWright[Feature]

### ISSUE THREAD

**Issue by ashishagrawal-TU** [microsoft/playwright#28006]
I didn't find if Island browser support is available in PlayWright(PW). if this is not there can you please tell me if this can also be added in browser support list?
This may not be specific to language but I am looking for Island browser support for Java language.

**Comment by mxschmitt** [comment]
It says:

> Island looks, feels, and behaves just like the browser you’re on right now.
> It’s built on Chromium - the same open source project that powers Chrome,
> Edge, and most other mainstream browsers.

on their website. So setting `executablePath` should make it work out of the box:

```ts
import { defineConfig } from '@playwright/test';

export default defineConfig({
  projects: [
    {
      name: 'chromium',
      use: {
        ...devices['Desktop Chrome'],
        launchOptions: {
          executablePath: '/you/path'
        }
      }
    }
  ]
});
```

**Comment by ashishagrawal-TU** [comment]
I checked the details for setExecutablePath [Path](optional)

Path to a browser executable to run instead of the bundled one. If executablePath is a relative path, then it is resolved relative to the current working directory. Note that Playwright only works with the bundled Chromium, Firefox or WebKit, **use at your own risk.** 
What is the meaning here for *use at your own risk.**. is there any issue from Security or stability point of view?

**Comment by mxschmitt** [comment]
We don't guarantee to run with third-party browsers but if they are a normal Chromium underneath then it will just work.

There no security implications, only stability ones.

**Comment by ashishagrawal-TU** [comment]
Thanks Max for the prompt query resolution! 
Will try it once application is ready for Island browser and let you know if I face any challenge.

**Comment by dgozman** [comment]
Closing because there is no action item for Playwright. If you still encounter problems, please file a new issue with a repro and link to this one.
